README.md

```diff
@@ -1 +1,132 @@
-# First-Capstone-Project
\ No newline at end of file
+# First-Capstone-Project
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b> Capstone Project README</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 First-Capstone-Project <a name="about-project"></a>
+
+
+This is the first capstone project in Microverse. The project is about an online web development course Udemy.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://reactjs.org/">JavaScript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Responsivness.
+- Interactions.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://muhammad0602.github.io/First-Capstone-Project/)
```

Comment:
Hello. I think the colors and font family are not requirements. Even though I did use Lato in my body but in some parts I changed it. 
There is a deployment problem because I deployed it via main branch and It shows you the content of the main branch.